### PR TITLE
Splitters UI tests rework

### PR DESCRIPTION
**Pull request by mmojzis**
{{pytest: cfme/tests/webui/test_splitter.py -v}}
Reworked pull splitter tests, because splitters ui was changed.
Deleted old splitters tests and methods.(broken_angular_select test is in same test file and it's not working so I told prt to test test_splitter.py only)
Added new splitters tests and methods.
Added webui folder in tests.
All the module changes are related to test_splitter only.

### CODE REVIEW

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit fffdae36317492dbf7ea21aed679b4136ccdfb31:

`cfme/tests/webui/test_splitter.py`:
- :heavy_exclamation_mark: Line 16:1: **W293** *blank line contains whitespace*

`docs/guides/tipsntricks.rst`:
- :grey_exclamation: Line 184: **INFO/1** *No directive entry for "automodule" in module "docutils.parsers.rst.languages.en".*

Please, rectify these issues :smirk: .

Commit inspection report:

**No commit flaws detected**.
You have multiple commits in the PR. When you want your PR to be merged, please squash them. Do not squash them when working on the pull request as this allows to see the progress!

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 1f2159acc2e931fb10553b3cee54120c183cb767:

`docs/guides/tipsntricks.rst`:
- :grey_exclamation: Line 184: **INFO/1** *No directive entry for "automodule" in module "docutils.parsers.rst.languages.en".*

Please, rectify these issues :smirk: .

Commit inspection report:

**No commit flaws detected**.
You have multiple commits in the PR. When you want your PR to be merged, please squash them. Do not squash them when working on the pull request as this allows to see the progress!

*CFME QE Bot*

File: cfme/tests/webui/test_splitter.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+@pytest.mark.meta(
+    blockers=[
+        BZ('1380443', unblock=lambda location: location != "bottlenecks")
+    ]
+)
+@pytest.mark.requirement('general_ui')
+@pytest.mark.tier(3)
+def test_pull_splitter_persistence(location):
+    # First we check hidden position splitter
```
Comment (mfalesni) [inline cfme/tests/webui/test_splitter.py]: Please add some comment with steps what does this test do.

**Comment by dajoRH** [comment]
Lint report for commit 7cd8516b93e47c34cec5a6199caea8d47656fc31:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
You have multiple commits in the PR. When you want your PR to be merged, please squash them. Do not squash them when working on the pull request as this allows to see the progress!

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 1bdb620fd3801e7a45f0660ca0e89e6f75663792:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
You have multiple commits in the PR. When you want your PR to be merged, please squash them. Do not squash them when working on the pull request as this allows to see the progress!

*CFME QE Bot*

File: cfme/tests/webui/test_splitter.py (hunk 1 above)
Comment (mmojzis) [inline cfme/tests/webui/test_splitter.py]: Done

**Comment by dajoRH** [comment]
Lint report for commit 0e09f9fff3f29d4607c4cb6ca8a7d2cf9e810302:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
You have multiple commits in the PR. When you want your PR to be merged, please squash them. Do not squash them when working on the pull request as this allows to see the progress!

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 42914bb7d60526f2a60c4517199d6007c7e63706:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
You have multiple commits in the PR. When you want your PR to be merged, please squash them. Do not squash them when working on the pull request as this allows to see the progress!

*CFME QE Bot*

File: cfme/web_ui/splitter.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+from cfme.fixtures import pytest_selenium as sel
+
+
+def pull_splitter_left():
+    sel.click("//span[@class='fa fa-angle-left']")
```
Comment (mmojzis) [inline cfme/web_ui/splitter.py]: Done (y)

**Comment by mshriver** [comment]
Commits need signed and PRT failures addressed before review. Thanks!

**Comment by dajoRH** [comment]
Lint report for commit 7dd0302b28f0144319bf2feb4c57ac0fedc7decf:

`cfme/web_ui/menu.py`:
- :bangbang: Line 395:21: **E126** *continuation line over-indented for hanging indent*
- :bangbang: Line 396:17: **E121** *continuation line under-indented for hanging indent*

Please, rectify these issues :smirk: .

Commit inspection report:

**No commit flaws detected**.
You have multiple commits in the PR. When you want your PR to be merged, please squash them. Do not squash them when working on the pull request as this allows to see the progress!

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit eaa09b1879f16b500bad8987eaa98c3f07687757:

`cfme/web_ui/menu.py`:
- :bangbang: Line 395:21: **E126** *continuation line over-indented for hanging indent*
- :bangbang: Line 396:17: **E121** *continuation line under-indented for hanging indent*

Please, rectify these issues :smirk: .

Commit inspection report:

**No commit flaws detected**.
You have multiple commits in the PR. When you want your PR to be merged, please squash them. Do not squash them when working on the pull request as this allows to see the progress!

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit eb80425ac9651eeceb80c7d09064400c7fdfb6a8:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit e1d328b5e3fb29564b88b5115fa80c249ca28793:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Pull request merged**